charts/aws-efs-csi-driver/Chart.yaml

```diff
@@ -2,8 +2,8 @@ apiVersion: v1
 appVersion: "1.2.0"
 name: aws-efs-csi-driver
 description: A Helm chart for AWS EFS CSI Driver
-version: 1.2.2
-kubeVersion: ">=1.17.0-0"
+version: 1.3.0
+kubeVersion: ">=1.18.0-0"
```

Comment:
what change is requiring 1.18?

Reply:
oh never mind, I see it's because of v1 csidriver in the changelog.

This complicates matters because technically the driver works with 1.17+ but now the primary install method requires 1.18+.

looks like ebs did the same change though so...https://github.com/kubernetes-sigs/aws-ebs-csi-driver/pull/836

Reply:
They did but I don't think it was handled correctly there.  They didn't change the kube version dependency in the chart.  I don't think it requires that the chart only work on 1.18+